src/Result/InvoicePayment.php

```diff
@@ -29,4 +29,18 @@ public function getStatus(): string
         $data = $this->getData();
         return $data['status'];
     }
+
+    public function getTransactionId(): string
+    {
+        $data = $this->getData();
+        $id = $data['id'];
+        $parts = explode('-', $id);
+        return $parts[0];
+    }
+
+    public function getReceivedTimestamp(): int
```

Comment:
I would suggest adding a simplified version of the comment as PHPDoc. Seems like relevant information to know (in case someone didn't read the full `README.md`)

Reply:
Hmm yeah, this could get tedious, but I do like idiot proof things.